app/jobs/application_submitted_sms_job.rb

```diff
@@ -3,7 +3,7 @@ def perform(export:)
     export.execute do |snap_application|
       Sms.new(snap_application).deliver_application_submitted_message
 
-      "Sent SMS to #{snap_application.phone_number}"
+      logger.info("Sent SMS to #{snap_application.phone_number}")
```

Comment:
nice! I find this much more intuitive with `logger.info` than just having the string

Reply:
Yea, I was really happy how passing the logger down into the Driver allowed me to backfill a more explicit way of storing information in the export.